README.md

```diff
@@ -1,4 +1,4 @@
-# Revenge of Task Manager
+# Task Manager Redux
```

Comment:
Do we want to change this to be `Task Manager` since they have no idea this is version two 👍